src/lib/integrations/daoStack/arc/schemes.ts

```diff
@@ -111,13 +111,23 @@ export const schemeDefinitions: SchemeDefinition[] = [
     permissions: "0x00000000" /* no permissions */,
     daoCanHaveMultiple: false,
     params: [
+      {
+        typeName: "beneficiary",
+        valueType: "Address",
+        displayName: "Beneficiary (keep empty to set it to the DAO)",
+        description:
+          "The beneficiary payment address. Keep this field empty to use the DAO as beneficiary.",
+        defaultValue: "",
+        optional: true,
+      },
       {
         typeName: "token",
         valueType: "Address",
-        displayName: "Pay Token",
+        displayName: "Pay Token (keep empty to use the DAO's native token)",
         description:
-          "The ERC20 token to pay for register or promotion, an address.",
-        defaultValue: "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359", // DAI. TODO: make this show up nicely in th UI
+          "The ERC20 token to pay for register or promotion, an address. Keep this field empty to use tha DAO's notive token.",
```

Comment:
"The address of an ERC20 token to pay for register or promotion. Keep this field empty to use the DAO's native token."